src/app/components/ArticlesList/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Article from '../../components/Article';
+
+import createUuidv4 from '../../utils/createUuidv4';
+
+import './scss/index.scss';
+
+const ArticlesList = (props) => {
```

Comment:
Please apply props destructuring for each component.

Reply:
Done.